extensions/amp-facebook/0.1/test/test-amp-facebook.js

```diff
@@ -20,7 +20,9 @@ import {adopt} from '../../../../src/runtime';
 
 adopt(window);
 
-describe('amp-facebook', () => {
+describe('amp-facebook', function() {
```

Comment:
This is fine, but I think we should consider, uhm, reducing the test coverage here :)